### ISSUE TITLE
[Suggestion] Improved handling of "Not interested" and "Follow" lists

### ISSUE THREAD

**Issue by r6d2** [jshackles/Enhanced_Steam#847]
# TL;DR
My suggestion is to make the *Not interested* and *Follow* attributes to work much like the Wishlist within Steam browser, aided by EnhancedSteam. This means adding the same sort/select options of the wishlist tab to the follow list tab, and creating or emulating a Not interested tab.

## Long version
Steam + EnhancedSteam provide together a very nice way to deal with your wishlist and with search pages, particularly during sales, allowing you to filter, sort by price, discount, etc..

Unfortunately, Valve has precluded the user of a good interface to the *Not interested* list. In fact, it does not provide a visible list at all. Some workarounds exist, like shown [on this thread] (https://steamcommunity.com/discussions/forum/1/618459109268261813/), but they are cumbersome and time consuming.

For the *Follow* list support is a little better within this [Steam browser page](http://steamcommunity.com/my/followedgames/).

Provided that EnhancedSteam has already access to each game flag like *Library*, *Wishlist*, *Gift* and *Not Interested*, I presume it may also access the *Follow* attribute of a game.

My suggestion is to make the *Not interested* and *Follow* attributes to work much like the Wishlist within Steam browser, aided by EnhancedSteam. This means adding the same sort/select options of the **Wishlist tab** to the **Follow list tab**, and creating or emulating a **Not interested tab**.

For instance, you could do things like these:
* List all *Not interested* games with offers sorted by price or discount. Why? Because even if you once felt don't want a game you may want to buy it for gifting or trading, particularly if it's on those less than a dollar sales. You may even end up buying it for you if you've changed your mind.

I use the *Follow* attribute to kind of soft-wishlist a game, meaning *"I'm not actively looking for this, but maybe I will in the future"*. Sometimes I *move* things from *Not interested* to *Follow*, or from *Follow* to *Wishlist*, and vice versa.

Also, from this standpoint, a hide "Items you follow" check mark just below "Items marked not interested" on the search page would be very useful to check only those games you've never checked before.

I don't know how difficult to implement all this is, but would really like feedback on the idea.

There's also another suggestion I was going to make, but I've seen its been asked already, although there're no comments on the threads mentioning it so I don't know if they are being considered. It's [here](https://github.com/jshackles/Enhanced_Steam/issues/619] and [here](https://github.com/jshackles/Enhanced_Steam/issues/747).

Best regards and congrats on such a useful software,

r6d2.

**Comment by ivanlonel** [comment]
This! So much this!

Also, it would be great to have tagging/highlighting for followed and not interested games.

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
